scripts/build.js

```diff
@@ -23,7 +23,7 @@ database.connect(function (db) {
         // builder basic pokemon details
         const builder = require(__base + 'app/controllers/filler/' + collection);
         builder.insertToDb(function () {
-            return;
+            process.exit();
```

Comment:
@jonas-he & @fabe85  I think we can use `process.exit` at the end since `return` is anyways not working. So I think we can merge this.